daemon/api_quotas.go

```diff
@@ -100,9 +100,13 @@ func createQuotaValues(grp *quota.Group) *client.QuotaValues {
 	}
 	if grp.JournalLimit != nil {
 		constraints.Journal = &client.QuotaJournalValues{
-			Size:       grp.JournalLimit.Size,
-			RateCount:  grp.JournalLimit.RateCount,
-			RatePeriod: grp.JournalLimit.RatePeriod,
+			Size: grp.JournalLimit.Size,
+		}
+		if grp.JournalLimit.RateEnabled {
```

Comment:
with this new code don't we also need a test where RateEnabled is false?

Reply:
Good point, it seems we had no test that hit this.